### PR TITLE
new athena query for support

### PR DESCRIPTION
**Pull request by ben851**
# Summary | Résumé

Adding an alb query for athena that lets you look up an ip address based on the URL between a date range.

## Related Issues | Cartes liées

N/A - Ad-hoc support request

# Test instructions | Instructions pour tester la modification

Terraform applies - verify the query exists in staging.

# Release Instructions | Instructions pour le déploiement

None.

# Reviewer checklist | Liste de vérification du réviseur

- [ ] This PR does not break existing functionality.
- [ ] This PR does not violate GCNotify's privacy policies.
- [ ] This PR does not raise new security concerns. Refer to our GC Notify Risk Register document on our Google drive.
- [ ] This PR does not significantly alter performance.
- [ ] Additional required documentation resulting of these changes is covered (such as the README, setup instructions, a related ADR or the technical documentation).

> ⚠ If boxes cannot be checked off before merging the PR, they should be moved to the "Release Instructions" section with appropriate steps required to verify before release. For example, changes to celery code may require tests on staging to verify that performance has not been affected.

### CODE REVIEW

File: aws/common/sql/alb_ip_lookup_by_url.sql.tmpl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+select * from alb_logs
```
Comment (sastels) [inline aws/common/sql/alb_ip_lookup_by_url.sql.tmpl:1]: 🤔 We could cut down / reorder the fields here to just have the most relevant... like time, request_url, client_ip, maybe request_verb?

**Review by sastels — CHANGES_REQUESTED** [review]
Just one request 🙏

File: aws/common/sql/alb_ip_lookup_by_url.sql.tmpl (hunk 1 above)
Comment (ben851) [reply aws/common/sql/alb_ip_lookup_by_url.sql.tmpl:1]: I knew this request was coming when I submitted the PR 😆  I'll do that just for you

**Review by sastels — APPROVED** [review]
Awesome!

**Pull request merged**